zebra/zebra_netns_notify.c

```diff
@@ -212,6 +212,12 @@ static int zebra_ns_notify_read(struct thread *t)
 			continue;
 		if (event->mask & IN_DELETE)
 			return zebra_ns_delete(event->name);
+		if ((char *)event->name + event->len
```

Comment:
name isn't a pointer, but rather an array.  for clarity and consistency how about:
  if (&event->name[event->len] >= &buf[len]) {